sdk/tables/azure-data-tables/tests/test_table_batch_cosmos.py

```diff
@@ -0,0 +1,691 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
```

Comment:
Given the lack of diff - are you able to call out any places where you had to alter the tests (ignoring the preparers) to get them to work with Cosmos? (Unless they're things we're already aware of)